src/functionalTest/java/uk/gov/hmcts/reform/juddata/cameltest/JrdBatchTestValidationTest.java

```diff
@@ -193,50 +188,11 @@ void testParentOrchestrationJsrExceedsThresholdAuditTest() throws Exception {
             containsString("Jsr exception exceeds threshold limit"));
     }
 
-    @Test
-    void testLeafFailuresInvalidHeaderContractsRollback() throws Exception {
-        uploadBlobs(jrdBlobSupport, archivalFileNames, true, file);
-        uploadBlobs(jrdBlobSupport, archivalFileNames, false, LeafIntegrationTestSupport.file_error);
```

Comment:
why removed this? we still need rest 2 leafs

Reply:
This is a change w.r.t previous PR.